x-pack/platform/packages/shared/kbn-ai-assistant/src/chat/welcome_message_knowledge_base.tsx

```diff
@@ -51,33 +63,30 @@ export function WelcomeMessageKnowledgeBase({
     }
   }, [knowledgeBase.status.value?.ready, previouslyNotInstalled, reset]);
 
-  // When the knowledge base is installed, stop checking for install status
+  // When the knowledge base is ready, stop polling for status
   useEffect(() => {
-    if (!checkForInstallStatus && knowledgeBase.status.value?.ready) {
-      setCheckForInstallStatus(false);
+    if (knowledgeBase.status.value?.ready) {
+      setPollKnowledgeBaseStatus(false);
     }
-  }, [checkForInstallStatus, knowledgeBase.status.value?.ready]);
+  }, [pollKnowledgeBaseStatus, knowledgeBase.status.value?.ready]);
 
-  // Check for install status every 5 seconds
+  // poll for knowledge base status every 5 seconds
   useInterval(
     () => {
       knowledgeBase.status.refresh();
     },
-    checkForInstallStatus ? 5000 : null
+    pollKnowledgeBaseStatus ? 5000 : null
   );
 
-  const handleRetryInstall = async () => {
-    setCheckForInstallStatus(true);
+  // gets called if there was an error previously during install or user has a preconfigured connector
+  // and is first time installing
+  const handleInstall = async () => {
     setIsPopoverOpen(false);
-
-    await knowledgeBase.install().then(() => {
-      setCheckForInstallStatus(false);
```

Comment:
don't stop checking kb status after `/setup` has returned.  model may not be ready.